src/comments.js

```diff
@@ -409,6 +411,24 @@ function handleObjectProperty(enclosingNode, precedingNode, comment) {
   return false;
 }
 
+function handleTemplateLiteralComments(enclosingNode, comment, options) {
+  if (
+    enclosingNode &&
+      enclosingNode.type === "TemplateLiteral"
+  ) {
+    const expressionIndex = findExpressionIndexForComment(
```

Comment:
@vjeux Sure! First thing, we need to find the correct expression in the template literal to attach the comment.